docs/pages/application-access/getting-started.mdx

```diff
@@ -77,19 +77,32 @@ Install Teleport on the host where you will run the Teleport Application Service
 
 Now start Teleport and point it to the application endpoint:
 
-```code
-$ sudo teleport app start \
-  --name=grafana \
-  --token=/tmp/token \
-  --uri=http://localhost:3000 \
-  --auth-server=https://teleport.example.com:3080
+This can be done by adding the application service to your teleport config, typically located at `/etc/teleport.yaml`.
+
+Add these lines to your config:
+
+```yaml
+app_service:
+  enabled: "yes"
+  apps:
+  - name: "grafana"
+    uri: "https://grafana.example.com:3000"
```

Comment:
Hmm, two of the reasons you might want to put teleport in front of Grafana are:

1. Your grafana instance is not publicly available and you want to enable connectivity to it
2. You don't have TLS certs for your Grafana deployment

The previous use of http and a localhost address showed both of those.

With this update, the URI looks like a publicly available, TLS-secured endpoint. Not necessarily wrong, but might leave the reader wondering what the point is.